roles/nv_gpu_install_from_commit/files/operator_helper_image_builder.yml

```diff
@@ -51,6 +51,7 @@ spec:
             git \
             bzip2 \
             xz \
+            go \
```

Comment:
This one is interesting

Reply:
`go` is called inside the `Makefile`, I install it to mute a warning at the beginning of `make docker-build`